bin/compile

```diff
@@ -66,6 +66,18 @@ if [[ -f "${ENV_DIR}/KEYCLOAK_PROVIDERS" ]]; then
   IFS=' ' 
 fi
 
+step "Adding providers from local build"
+KEYCLOAK_CUSTOM_PROVIDER_DIR="dev/providers"
+if [[ -f "${ENV_DIR}/KEYCLOAK_CUSTOM_PROVIDER_DIR" ]]; then
+  KEYCLOAK_CUSTOM_PROVIDER_DIR=$(cat "${ENV_DIR}/KEYCLOAK_CUSTOM_PROVIDER_DIR")
+fi
+if [[ -d "${BUILD_DIR}/${KEYCLOAK_CUSTOM_PROVIDER_DIR}" ]]; then
+  jar_file=$(find "${BUILD_DIR}/${KEYCLOAK_CUSTOM_PROVIDER_DIR}"/*.jar | head -n 1)
```

Comment:
Note that we are picking up the first jar present in the KEYCLOAK_CUSTOM_PROVIDER_DIR directory